src/components/DocsNav.tsx

```diff
@@ -53,51 +42,40 @@ const CardLink = ({ docData, isPrev, contentNotTranslated }: CardLinkProps) => {
   const { t } = useTranslation("page-developers-docs")
   const { isRtl } = useRtlFlip()
 
-  const xPadding = isPrev ? { ps: "0" } : { pe: 0 }
+  const xPaddingClass = isPrev ? "ps-0" : "pe-0"
 
   return (
-    <LinkBox
-      as={Flex}
-      alignItems="center"
-      w="full"
-      flex="1"
-      h="82px"
-      bg="background.base"
-      border="1px"
-      borderColor="border"
-      borderRadius={1}
-      justify={isPrev ? "flex-start" : "flex-end"}
+    <BaseLink
+      href={docData.href}
+      className={cn(
+        "flex w-full flex-1 items-center no-underline",
+        "h-[82px] rounded-[1px] border bg-background",
+        isPrev ? "justify-start" : "justify-end"
+      )}
+      rel={isPrev ? "prev" : "next"}
+      customEventOptions={{
+        eventCategory: "next/previous article DocsNav",
+        eventAction: "click",
+        eventName: isPrev ? "previous" : "next",
+      }}
     >
-      <Box textDecoration="none" p={4} h="100%" order={isPrev ? 0 : 1}>
+      <div className={cn("h-full p-4", isPrev ? "order-[0]" : "order-1")}>
         <Emoji
           text={isPrev ? ":point_left:" : ":point_right:"}
           className={cn(
             "text-5xl",
             !contentNotTranslated && isRtl ? "-scale-x-100" : ""
           )}
         />
-      </Box>
-      <TextDiv {...xPadding} {...(!isPrev && { textAlign: "end" })}>
-        <Text textTransform="uppercase" m="0">
-          {t(isPrev ? "previous" : "next")}
-        </Text>
-        <LinkOverlay
-          as={BaseLink}
-          href={docData.href}
-          textAlign={isPrev ? "start" : "end"}
-          rel={isPrev ? "prev" : "next"}
-          onClick={() => {
-            trackCustomEvent({
-              eventCategory: "next/previous article DocsNav",
-              eventAction: "click",
-              eventName: isPrev ? "previous" : "next",
-            })
-          }}
-        >
+      </div>
+      <TextDiv className={cn(xPaddingClass, !isPrev ? "text-end" : "")}>
```

Comment:
to simplify, we could do this since we are not reusing `xPaddingClass`.
```suggestion
      <TextDiv className={cn(isPrev ? "ps-0" : "pe-0 text-end")}>
```